### ISSUE TITLE
Improve Video Playback

### ISSUE THREAD

**Issue by sschueller** [sschueller/peertube-android#6]
Currently video plays via WebTorrent only which is slow. There are also some player issues (rotating device) but those are player related.

From what I understand if we were to use WebRTC video could start playing right away.

I could use some help with this as I am not very familiar with WebRTC especially on Android.

**Comment by Chocobozzz** [comment]
I don't think you could use WebRTC in the application, but it's weird the video playing is slow. Does your torrent library supports WebSeed? If an Android application only uses TCP/UDP and WebSeed, I think it should be fine (even if it does not support WebRTC peers).

**Comment by sschueller** [comment]
I am not sure if the library supports WebSeed. I am currently using https://github.com/TorrentStream/TorrentStream-Android but I will look into it or maybe switch to another library. Thanks
